students/srepking/Lesson03/mailroom.py

```diff
@@ -77,9 +77,10 @@ def report():
         response = input('Type \'1\' to Send a Thank You, \'2\' to Create a Report, or \'3\' to quit?')
         if response == '1':
             thank_you()
-        if response == '2':
+        elif response == '2':
             report()
-        if response == '3':
+        else:
+            response == '3'
```

Comment:
Hi Natasha, 
No, I didn't. I sent an email in response to this, though I see now I can respond from within Github! This is what I intended. Though I am a bit unsure why I can't just use three 'if' statements instead of 'elif'. Seems like it does the same thing?
       if response == '1':
            thank_you()
        elif response == '2':
            report()
        elif response == '3':
            print('Goodbye!')
            break